test/parallel/test-zlib-random-byte-pipes.js

```diff
@@ -154,23 +150,9 @@ const gunz = zlib.createGunzip();
 
 inp.pipe(gzip).pipe(gunz).pipe(out);
 
-inp.on('data', function(c) {
-  console.error('inp data', c.length);
-});
-
-gzip.on('data', function(c) {
-  console.error('gzip data', c.length);
-});
-
-gunz.on('data', function(c) {
-  console.error('gunz data', c.length);
-});
-
-out.on('data', function(c) {
-  console.error('out data', c.length);
-});
-
-out.on('data', common.mustCall(function(c) {
-  console.error('hash=%s', c);
+inp.on('data', common.mustCall(4));
```

Comment:
Was a bit afraid of that. Looking at it, the on('data') events here look entirely unnecessary.